tests/models/bridgetower/test_modeling_bridgetower.py

```diff
@@ -0,0 +1,362 @@
+class BridgeTowerModelTest(ModelTesterMixin, unittest.TestCase):
+    all_model_classes = (
+        (BridgeTowerModel, BridgeTowerForImageAndTextRetrieval, BridgeTowerForMaskedLM) if is_torch_available() else ()
+    )
+
+    is_training = False
+    test_headmasking = False
+    test_pruning = False
+    test_torchscript = False
+    test_resize_embeddings = False
+    has_attentions = False
+
+    # function to extract meaningful tensor from output per different model_class
+    def extract_output(self, outputs, model_class):
+        return outputs["pooler_output"] if model_class == "BridgeTowerModel" else outputs["logits"]
+
+    def setUp(self):
+        self.model_tester = BridgeTowerModelTester(self)
+        self.config_tester = ConfigTester(self, config_class=BridgeTowerConfig, hidden_size=37, vocab_size=50265)
+
+    def test_config(self):
+        self.config_tester.run_common_tests()
+
+    def test_model(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_model(*config_and_inputs)
+
+    def test_for_image_and_text_retrieval(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_for_image_and_text_retrieval(*config_and_inputs)
+
+    def test_for_masked_language_modeling(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_for_masked_language_modeling(*config_and_inputs)
+
+    @slow
+    def test_model_from_pretrained(self):
+        for model_name in BRIDGETOWER_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
+            model = BridgeTowerModel.from_pretrained(model_name)
+            self.assertIsNotNone(model)
+
+    def test_save_load(self):
+        config, input_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+            with torch.no_grad():
+                outputs = model(**input_dict)
+
+            out_2 = self.extract_output(outputs, model_class.__name__)
+            out_2 = out_2.cpu().numpy()
+            out_2[np.isnan(out_2)] = 0
+
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                model.save_pretrained(tmpdirname)
+                model = model_class.from_pretrained(tmpdirname)
+                model.to(torch_device)
+                with torch.no_grad():
+                    after_outputs = model(**input_dict)
+
+                # Make sure we don't have nans
+                out_1 = self.extract_output(after_outputs, model_class.__name__)
+                out_1 = out_1.cpu().numpy()
+                out_1[np.isnan(out_1)] = 0
+                max_diff = np.amax(np.abs(out_1 - out_2))
+                self.assertLessEqual(max_diff, 1e-5)
+
+    def test_determinism(self):
```

Comment:
What's the reason this test is overridden?

Reply:
The extraction of meaningful tensors from output of BridgeTower was different, thus the test was overridden. In our latest commit, thanks for the suggestion, this test is no longer overridden.